src/app/api/images/og/assets/shared.tsx

```diff
@@ -7,7 +7,7 @@ export const LogoPill = () => {
 
   return (
     <div tw="flex">
-      <div tw="flex flex-row items-center border border-line rounded-full px-[26px] py-[14px]">
```

Comment:
@mcgingras I reverted this change because the styles were not showing up in the generated image. We can debug this when we need to offer more customizations.

Reply:
No problem! This is pretty specific part of the app so I think it's okay if it's default themed